docs/api/api_blueprint.apib

```diff
@@ -17483,7 +17483,7 @@ Create a new Discount Code for event. (Only Admin)
     + sort (optional, string, `code`) - Sort the resources according to the given attribute in ascending order. Append '-' to sort in descending order.
     + filter (optional, string, `[]`) - Filter according to the flask-rest-jsonapi filtering system. Please refer: http://flask-rest-jsonapi.readthedocs.io/en/latest/filtering.html for more.
 
-### Create Ticket Discount Code [POST]
+### Create Ticket Discount Code [POST /v1/discount-codes{?page%5bsize%5d,page%5bnumber%5d,sort,filter}]
```

Comment:
This change would also be needed for `Create Access Code`

Reply:
@kushthedude Why?

Reply:
For the access code, Only `POST` is written without the exact URL .

Reply:
@kushthedude Read API Blueprint documentation. And before giving reviews, make sure you understand the issue and the solution implemented.